.pre-commit-config.yaml

```diff
@@ -14,52 +14,53 @@ repos:
     -   id: check-merge-conflict
 
 - repo: https://github.com/Riverside-Healthcare/djLint
-  rev: v1.34.1
+  rev: v1.36.4
   hooks:
     - id: djlint-reformat
       args: ["--no-function-formatting"]
     - id: djlint
 
 -   repo: https://github.com/Yelp/detect-secrets
-    rev: v1.4.0
+    rev: v1.5.0
     hooks:
     -   id: detect-secrets
         args: ['--baseline', '.secrets.baseline']
 
 -   repo: https://github.com/pycqa/flake8
-    rev: 7.0.0
+    rev: 7.2.0
     hooks:
     - id: flake8
 
 -   repo: https://github.com/psf/black
-    rev: 24.2.0
+    rev: 25.1.0
     hooks:
     - id: black
       language_version: python3
 
 -   repo: https://github.com/pycqa/isort
-    rev: 5.13.2
+    rev: 6.0.1
     hooks:
       - id: isort
         name: isort (python)
         args: ["--profile", "black", "--filter-files"]
 
 -   repo: https://github.com/PyCQA/bandit
-    rev: 1.7.7
+    rev: 1.8.3
     hooks:
     -   id: bandit
         args: ["-c", "pyproject.toml", "--exclude", "node_modules/*", "-r", "."]
 
--   repo: local
+-   repo: https://github.com/python-poetry/poetry
```

Comment:
what's this change? No more safety? This just generates a requirements.txt file but does nothing with itfro what I can tell?